packages/cli/src/commands/info.js

```diff
@@ -20,69 +22,108 @@ export default class InfoCommand extends Command {
     version: '1.0.0',
     args: [],
     options: [
-      [ '--debug', 'include os versions in output' ],
+      ['--debug', 'include os versions in output']
     ],
   }
 
   constructor() {
     super(InfoCommand.definition)
   }
 
-  action(options, command) {
+  async action(options, command) {
     if (options.debug) {
-      console.debug('[CLI] Command \'%s\' called with options %o', this.name(), options)
+      console.debug(
+        '[CLI] Command \'%s\' called with options %o',
+        this.name(),
+        options
+      )
     } else {
       console.debug('[CLI] Command \'%s\' called', this.name())
     }
 
+    let versionFile = fs.readFileSync(VERSION_FILE, { encoding: 'utf8' })
+    try {
+      versionFile = JSON.parse(versionFile)
+    } catch (error) {
+      console.warn(
+        `This project was generated with quire-cli prior to version 1.0.0.rc-8, the quire version file does not contain specific version information.`
+      )
+      fs.writeFileSync(VERSION_FILE, JSON.stringify({}))
```

Comment:
We might add to the message "updating version file to new format." so that users do not start asking how to fix that themselves... something like,
```suggestion
      console.warn(`Updating the quire project version file to new format.`)
      fs.writeFileSync(VERSION_FILE, JSON.stringify({}))
```